### DISCUSSION TITLE
Running ImageMagick Multicrop script from .Net

### DISCUSSION THREAD

**Discussion by SimonKravis** [ImageMagick/ImageMagick#2025] [Help]
I'd like to run [Fred Weinhaus' multicrop script](http://www.fmwconcepts.com/imagemagick/multicrop/index.php) from a C# application using .NET. Can I do this using the ImageMagick .Net object or by running ImageMagick from the command line?

**Comment by fmw42** [comment]
It has not been converted to Magick.NET. The script is Unix bash shell script. So only runs on Unix-like systems. You can use Windows 10 Unix or Windows with Cygwin to run the script. 

But the basis of the script is -connected-components. Threshold your image or flood fill it so that you get a binary mask. Process the mask with -connected-components to get the bounding boxes. Then use the bounding boxes to crop your image.
